### PR TITLE
[DEV-1539] Align "Go to webinar" buttons

### PR DESCRIPTION
**Pull request by jeremygordillo**
#### List of Changes

Update webinar card styles

#### Motivation and Context

image

#### How Has This Been Tested?

Manually. Go to [webinar page](https://developer.pagopa.it/webinars) and check section "Webinar passati"
#### Screenshots (if appropriate):
image

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by MikeAtUqido — APPROVED** [review]

File: apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx [hunk 1]
```diff
@@ -49,12 +50,14 @@ const WebinarListItem = ({ webinar }: WebinarListItemProps) => {
...
       )}
-      <Typography variant='h6'>{webinar.title}</Typography>
-      <Stack mt={2} direction='row' alignItems='center' color='primary.main'>
-        <LinkButton
-          href={`/webinars/${webinar.slug}`}
-          label={t('webinar.goToWebinar')}
-        />
+      <Stack justifyContent='space-between' flexGrow={1}>
+        <Typography variant='h6'>{webinar.title}</Typography>
+        <Stack mt={2} direction='row' alignItems='center' color='primary.main'>
```
Comment (marcobottaro) [inline apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx:55]: Is this `Stack` necessary?

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx (hunk 1 above)
Comment (jeremygordillo) [reply apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx:55]: Yes, without this we do not achieve the desired behavior

Comment (marcobottaro) [reply apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx:55]: I think that you don't need to use a `Stack` if it has only one child. A `Box` is enough

**Review by marcobottaro — CHANGES_REQUESTED** [review]

File: apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx (hunk 1 above)
Comment (jeremygordillo) [reply apps/nextjs-website/src/components/molecules/WebinarListItem/WebinarListItem.tsx:55]: I misunderstood which `<Stack/>` you were referring to, so I proceed to update the `<Stack/>` on line 55

**Review by marcobottaro — APPROVED** [review]

**Pull request merged**